source/includes/_resource.collections.md

```diff
@@ -895,6 +946,141 @@ HEAD 204
+```
+
+> Example Request
+
+```shell
+$ curl -X PUT "https://api.vhx.tv/collections/1/items/32" \
+  -d '{"position": 3}' \
+  -u o3g_4jLU-rxHpc9rsoh3DHfpsq1L6oyM:
+```
+
+> Example Response
+
+```json
+{
+  "_links": {
+    "self":  { "href": "https://api.vhx.tv/videos/1" },
+    "files": { "href": "https://api.vhx.tv/videos/1/files" }
+  },
+  "_embedded": {
+    "files": [
+      {
+        "_links": {
+          "self":  { "href": "https://api.vhx.tv/videos/1/files?quality=adaptive&format=m3u8" }
+        },
+        "quality": "adaptive",
+        "format": "m3u8",
+        "method": "hls",
+        "size": {
+          "bytes": 1073741824,
+          "formatted": "1 GB"
+        },
+        "mime_type": "application/x-mpegURL"
+      }
+    ]
+  },
+  "id": 1,
+  "title": "My Video",
+  "description": "My video description.",
+  "status": "uploaded",
+  "duration": {
+    "seconds": 7200,
+    "formatted": "02:00:00"
+  },
+  "thumbnail": {
+    "small": "https://cdn.vhx.tv/assets/thumbnails/default-small.png",
+    "medium": "https://cdn.vhx.tv/assets/thumbnails/default-medium.png",
+    "large": "https://cdn.vhx.tv/assets/thumbnails/default-large.png",
+    "source": "https://cdn.vhx.tv/assets/thumbnails/original.jpg"
+  },
+  "tracks": {
+    "subtitles": [
+      {
+        "_links": {
+            "srt": { "href": "https://cdn.vhx.tv/file.srt" },
+            "vtt": { "href": "https://cdn.vhx.tv/file.vtt" }
+        },
+        "label": "English",
+        "srclang": "en",
+        "kind": "subtitles"
+      }
+    ]
+  },
+  "advertising": {
+    "_links": {
+      "tag": { "href": "https://static.vhx.tv/vmap.xml" }
+    },
+    "client": "vmap",
+    "provider": "dfp",
+    "custom_params": {
+      "foo": ["some", "value"],
+      "bar": ["other", "thing"]
+    }
+  },
+  "metadata": {
+    "director": "Brad Pitt",
+    "writers": ["Foo Bar", "Bar Foo"],
+    "release_year": 2017,
+    "series_name": "Series Name",
+    "season_name": "Season 1",
+    "season_number": 1,
+    "episode_number": 5,
+    "custom_icon": {
+      "small": "https://vhx.imgix.net/site/assets/1231.jpg?fit=clip&h=135&w=240",
+      "medium": "https://vhx.imgix.net/site/assets/1231.jpg?fit=clip&h=360&w=640",
+      "large": "https://vhx.imgix.net/site/assets/1231.jpg?fit=clip&h=720&w=1280",
+      "source": "https://vhx.imgix.net/site/assets/1231.jpg"
+    },
+    "advertising_keywords": []
+  },
+  "plans": ["public", "free", "standard"],
+  "time_available": "2014-03-01T00:00:00Z",
+  "time_unavailable": "2014-04-01T00:00:00Z",
+  "files_count": 5,
+  "finishes_count": 1204,
+  "created_at": "2014-02-25T20:19:30Z",
+  "updated_at": "2014-02-25T20:19:30Z",
+  "item_id": 32,
+  "position": 3
+}
+```
+
+<section class="text-2 contain margin-bottom-medium">
+  <p>Updates the position of an item in a collection. The <code>:item_id</code> refers to the <code>item_id</code> value present in the response from the List all Items endpoint.</p>
```

Comment:
Does this really only update the position of the collection? Can't we update any other attributes through this endpoint?

Reply:
It updates only the position: https://github.com/vhx/crystal/blob/2326dffd408395828079efa50ed192a049f8061f/app/controllers/api/collection_items_controller.rb#L46